setup.cfg

```diff
@@ -0,0 +1,13 @@
+[check-manifest]
+ignore =
+    */test
+    */test/*
+    .*
+    node_modules
+
+[pep257]
+ignore = D202
+explain = true
+
+[yapf]
+based_on_style = pep8
```

Comment:
I'm happy to adopt it then.

I _would_ like us at some point to move towards "convergence" with our Python lint and formatting etc tools. We have style guides that say one thing, then actual practice that does another, then linters that do another and a formatter that does something else (fairly sure there are some contradictions between all of these). We have an unfortunate difference between what lint tools `make lint` runs and what we can get running with GitHub integration (since Landscape sucks, but Hound doesn't support prospector or pylint).

It's not urgent but I think it's be nice to get this down